docs/VSSetup/about_VSSetup.md

```diff
@@ -0,0 +1,31 @@
+﻿# VSSetup
+## about_VSSetup
+
+# SHORT DESCRIPTION
+Enumerate and select instances of Visual Studio.
+
+# LONG DESCRIPTION
+Visual Studio 2017 introduced a new setup engine capable of installing multiple instances of Visual Studio and other products in the Visual Studio family. This module provides commands to enumerate those instances and select instances that meet your criteria. For example, in a development environment you might have a script that finds an instance of Visual Studio with the the Managed Desktop workload for writing projects targeting the .NET Framework. See below for more examples.
```

Comment:
Duplicate 'the' on the third line of this description: instance of Visual Studio with the **the** Managed